doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -429,7 +429,7 @@ Induction
 
    .. example::
 
-      .. coqtop:: reset all
+      .. coqdoc::
```

Comment:
Is a warning missing for some tactics in this chapter that they require loading the Stdlib?